code/game/objects/items/devices/traitordevices.dm

```diff
@@ -291,30 +297,41 @@
 	if(times_used < max_uses)
 		. += "<span class='notice'>[src] has [max_uses-times_used] charges left.</span>"
 
+/obj/item/batterer/process()
+	if(times_used)
```

Comment:
Would it be better if this was an early return, so we didn't also check for the `if(recharge_ticks >= 10)` that we have below?

Reply:
No, I only want it to start charging when it's used